helper/util.go

```diff
@@ -344,10 +358,18 @@ func BuildAndBroadcastMsgsWithCLI(cliCtx context.CLIContext, txBldr authTypes.Tx
 }
 
 // GetSignedTxBytes returns signed tx bytes
-func GetSignedTxBytes(cliCtx context.CLIContext, txBldr authTypes.TxBuilder, msgs []sdk.Msg) ([]byte, error) {
+func GetSignedTxBytes(cliCtx context.CLIContext,
+	txBldr authTypes.TxBuilder,
+	msgs []sdk.Msg,
+	testOpts ...*TestOpts,
+) ([]byte, error) {
 	// just simulate (useful for testing)
 	if cliCtx.Simulate {
-		return nil, nil
+		if len(testOpts) == 0 || testOpts[0].chainId == "" {
+			return nil, nil
+		}
+		txBldr := txBldr.WithChainID(testOpts[0].chainId)
```

Comment:
I don't think you need the assignment `:` here

Reply:
Removed [89c97b2](https://github.com/maticnetwork/heimdall/pull/1170/commits/89c97b2b0a4f76edcdba870ffa0e9521614c90cf).